.github/workflows/test.yml

```
@@ -507,13 +492,13 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
           submodules: recursive
 
-      - name: yarn-build
-        uses: ./.github/actions/yarn-build-with-cache
+      - name: pnpm-build
+        uses: ./.github/actions/pnpm-build-with-cache
         with:
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
```

actionlint (1.7.9)

input "ref" is not defined in action "PNPM Build with Cache" defined at "./.github/actions/pnpm-build-with-cache". available inputs are 

(action)